### DISCUSSION TITLE
Descriptions for pictures don't work

### DISCUSSION THREAD

**Discussion by SirSatorik** [TheLastBen/fast-stable-diffusion#2857] [Q&A] [answered]
Hi all! I have a problem when I start training the sd 1.5 model on google colab (fast-dreamboothy), it always doesn’t find the description and just writes “image” or “image image” in the upload field. What should I do? My friend said that this field should contain "image <description>". And the problem is not purely visual, but also the model does not work. Of course, I load the description and when I run the "Captions (Optional)" section, everything is displayed. I save all photos and descriptions like this:

- image_1.png
- image_1.txt
-
- image_2.png
- image_2.txt

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/108075233/a9996415-efb5-469d-935f-baa66e02dd1c)

**Comment by TheLastBen** [comment, marked as answer]
did you check the box "external captions"

**Reply by SirSatorik** [reply]
No, I didn’t install it, apparently because of this it didn’t work. I’ll check and write now, but thanks in advance!

**Reply by SirSatorik** [reply]
Thanks a lot
